RockWeb/Themes/NewSpring/assets/Lava/UI/card.lava

```diff
@@ -17,14 +17,6 @@
 {% assign cardlinkurl = cardlinkurl | Trim %}
 
 <div {% if id and id != empty %}id="{{ id }}"{% endif %} class="card position-relative panel panel-default overflow-hidden {% if alignment != null and alignment != '' %}text-{{ alignment }}{% endif %} rounded">
-    {% if cciid and cciid != empty %}
-        {% assign hasRights = cciid | HasRightsTo:'Edit','Rock.Model.ContentChannelItem' %}
-        {% if hasRights == 'true' %}
-            <div class="position-absolute top-zero right-zero bg-info bring-forward shadowed" style="border-bottom-left-radius: 8px;">
-                {[ button linkurl:'/page/344?contentItemId={{ cciid }}' linktext:'Edit This' buttontype:'info' buttonsize:'xs' linktarget:'_blank' ]}
-            </div>
-        {% endif %}
-    {% endif %}
```

Comment:
Is this button not needed anymore?

Reply:
I asked Frank and we're removing these from the site as they are preventing us from being able to utilize output cache since they were personalized.